website/src/components/Header/Header.tsx

```diff
@@ -1,12 +1,11 @@
-import { Box, Button, useColorMode } from "@chakra-ui/react";
+import { Box, Button, Text, useColorMode } from "@chakra-ui/react";
```

Comment:
It seems like in the head of this branch `useColorMode` is actually removed because it is unused. However I think this is the way we should do light dark switching.